### PR TITLE
Added information regarding impacts of frequent backups

### PR DESCRIPTION
**Pull request by apurvabhide17**
Version(s):
OADP 1.4.1
OCP 4.13 → OCP 4.16

Issue:
[OADP-1793](https://issues.redhat.com/browse/OADP-1793)

Link to docs preview:
https://80245--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html

QE review:
- [x] QE has approved this change.

Additional information:
Apart from adding content regarding frequent backups and its impact, I've added a minor fix in the orignal abstract and fixed the syntax for link - See [Scheduling backups using Schedule CR](https://docs.redhat.com/en/documentation/openshift_container_platform/4.12/html-single/backup_and_restore/index#oadp-scheduling-backups-doc)]

### CODE REVIEW

**Review by skopacz1 — COMMENTED** [review]
Left a few comments, otherwise looks good

**Review by weshayutin — APPROVED** [review]
/LGTM

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 1]
```diff
@@ -7,7 +7,12 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You back up applications by creating a `Backup` custom resource (CR). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+Frequent backups might consume storage on the backup storage location. Check the frequency of backups, retention time, and the amount of data of the persistent volumes (PVs) if using non-local backups, for example, S3 buckets.
+Since all taken backup remains until expired, also check the time to live (TTL) setting of the schedule.
+
+Note that that there might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:modules/oadp-release-notes-1-1-2.adoc#known-issues_oadp-release-notes[Known issues].
```
[the remark this answers is not in the document]
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:13]: thanks for explaining this :)

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 2]
```diff
@@ -7,7 +7,12 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You back up applications by creating a `Backup` custom resource (CR). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+Frequent backups might consume storage on the backup storage location. Check the frequency of backups, retention time, and the amount of data of the persistent volumes (PVs) if using non-local backups, for example, S3 buckets.
+Since all taken backup remains until expired, also check the time to live (TTL) setting of the schedule.
```
[the remark this answers is not in the document]
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc]: done, thanks

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc]: done, thanks

[the remark this answers is not in the document]
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc]: done, thanks

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 3]
```diff
@@ -7,7 +7,12 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
...
+
+Note that that there might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:modules/oadp-release-notes-1-1-2.adoc#known-issues_oadp-release-notes[Known issues].
+
+You can back up applications by creating a `Backup` custom resource (CR). For moreinformation, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
```
[the remark this answers is not in the document]
Comment (apurvabhide17) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc]: done, thanks

**Comment by apurvabhide17** [comment]
Thanks, @weshayutin and @skopacz1 for reviewing my PR.

**Comment by apurvabhide17** [comment]
Hello @skopacz1, could you please let me know why the **ci/prow/validate-portal** is failing even after updating the xref syntax as you suggested? 
Thanks in advance!

**Comment by amastbau** [comment]
we start the session with this note
and it is not makred as note
wanderring about that 
otherwize, LGTM

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 4]
```diff
@@ -7,7 +7,15 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You back up applications by creating a `Backup` custom resource (CR). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+Frequent backups might consume storage on the backup storage location. Check the frequency of backups, retention time, and the amount of data of the persistent volumes (PVs) if using non-local backups, for example, S3 buckets.
+Because all taken backup remains until expired, also check the time to live (TTL) setting of the schedule.
+
+[NOTE]
+====
+There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:xref:../../../backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1.adoc#known-issues_oadp-release-notes[Known issues].
```
Comment (skopacz1) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:15]: there's an extra xref here:
```suggestion
There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:../../../backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1.adoc#known-issues_oadp-release-notes[Known issues].
```

**Review by skopacz1 — COMMENTED** [review]
Sorry, I didn't see your ping from earlier. I found what I believe is the cause of your build failing. If it still fails after you address my comment, I can take another look!

**Comment by apurvabhide17** [comment]
Since there was an issue while squashing commits, I had to create a new [PR](https://github.com/openshift/openshift-docs/pull/81869). 
I'm closing this PR and the new one will be merged with all the changes that were reviewed in this PR

**Pull request closed without merging**